docs/tutorial.md

```diff
@@ -0,0 +1,620 @@
+# Tutorial
+
+This assumes that you plan to deploy workload clusters on AWS cloud, with
+Cluster API ("CAPI") as the cluster orchestration API provider.
+Also ensure you have set up a [workspace repository](#workspace-repository),
```

Comment:
I guess the link here will need to be changed as it currently is not working currently

Reply:
okay, can you take care of it as part of https://github.com/arlonproj/arlon/issues/311 ?